### ISSUE TITLE
Standard C++ header not found. Ubuntu 18.04

### ISSUE THREAD

**Issue by mica100** [MaskRay/ccls#450]
### Observed behavior

Use a small program that includes any standard C++ header like `<cstdlib>, <iostream>, <memory>`, but Visual Studio Code show "not found" and "std undeclared". Even the compiler can build this program without errors.

Visual Studio code show 3 problems from `ccls` on `Main.cpp`:
```
'iostream' file not found [1,10]
use of undeclared identifier 'std' [4,5]
use of undeclared identifier 'std' [4,36]
```

Sample code:
```
#include <iostream>
int main(int argc, char const *argv[]) {
    std::cout << "Hello world." << std::endl;
    return 0;
}
```

Using this CMake file:
```
project(testccls)
add_executable(testccls Main.cpp)
```

This is the `compile_commands.json` created by CMake:
```
[
{
  "directory": "/home/myuser/Projects/t1",
  "command": "/usr/bin/c++      -o CMakeFiles/testccls.dir/Main.cpp.o -c /home/myuser/Projects/t1/Main.cpp",
  "file": "/home/myuser/Projects/t1/Main.cpp"
}
]
```

**Workaround**: Add the standard include path to a project local .ccls file. E.g. `.ccls` content:
```
%compile_commands.json
-I/usr/include/c++/7/
```

### Expected behavior
CCLS knows the standard include directories. E.g. by asking the used compiler (from compile_commands.json). Header iostream should be found and not red underlined and hover says "file not found" and problem `undeclared std` should not exist.

### Steps to reproduce

1. Put the sample code in `Main.cpp`, the CMake code in `CMakeLists.txt`
2. run `cmake -DCMAKE_EXPORT_COMPILE_COMMANDS=ON`
3. Open Visual Code with ccls enabled: `code .`
4. Open Main.cpp and check if `#include <iostream>` and `std::` is red underlined and listed in the problem list.

### System information

* ccls version (`git describe --tags --long`): 0.20190314.1-20-gaf72b59d
* clang version: 8.0.0 (download clang+llvm-8.0.0-x86_64-linux-gnu-ubuntu-18.04)
* OS: Ubuntu 18.04.2 LTS
* Editor: VS Code 1.36.1
* Language client (and version): VS Code, ccls 0.1.26

**Comment by MaskRay** [comment]
https://github.com/MaskRay/ccls/wiki/FAQ#some-cc-headers-are-not-recognized
especially the `-isystem` subsection.

**Comment by mica100** [comment]
Thank you for the hint. I skipped this section in the FAQ because it started with "First, if you are Emacs company mode user ..." comment and I didn't realize that the whole section is also for other users.

I used c-index-test command and found following lines useful on my Ubuntu 18.04 machine to make VS-Code with ccls:
1. finding all standard headers
2. not red underlining the `std`-elements like `cout`, `endl` or `make_unique`, after including `<iostream>` and `<memory>`.

Here are the `-I` lines for the `.ccls` file:
```
-I/usr/include/c++/7
-I/usr/include/x86_64-linux-gnu/c++/7
```

Comment: to make c-index-test command happy it must also include your clang directory e.g. `/opt/clang-8/lib/clang/8.0.0/include` but this is known by ccls (if it's still existing as it was during compile time) so it's not needed additionally in `.ccls` file.

**Comment by MaskRay** [comment]
wiki/FAQ might be a bit outdated. I just added some more contents there.

> -I/usr/include/c++/7
>
> -I/usr/include/x86_64-linux-gnu/c++/7

The two options should not be necessary. Check the output of `clang -v -fsyntax-only -x c++ /dev/null`

> Comment: to make c-index-test command happy it must also include your clang directory e.g. /opt/clang-8/lib/clang/8.0.0/include

It is a known bug. `c-index-test -index-file local a.cc -resource-dir /opt/clang-8/lib/clang/8.0.0` but I think the bug should have been fixed in 9.0.0 (I thought it was fixed in 8.0.0 ...)
